handlers/java_handler/src/test/java/mercator/MavenUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package mercator;
+
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URL;
+
+public class MavenUtilsTest  {
+
+    @Test
+    public void testXMLCleanup () {
+        URL resource = this.getClass().getResource("/mercator/osio_3182_pom-expanded.xml");
+        assert resource != null;
+        try {
+            URI uri = resource.toURI();
+            assert uri != null;
+            File f = new File(uri);
+            File f2 = mercator.MavenUtils.removeDuplicateXMLTag(f);
+            Document d = mercator.MavenUtils.readFileAsDocument(f2);
+            assert d != null;
```

Comment:
I don't really like this test because you are basically testing whether method returned something. You are not actually checking whether you got what you expected.

Other than that - You are not actually running these tests.

Reply:
This is the very first test for `java_handler` overall. Sure I can add a better check that verifies the contends of the returned document.

Reply:
Updated `Makefile` to run test.

Reply:
This is not actually true. You can check tests written in `behave`.

Reply:
I meant there are on tests for `java_handler` code base itself using JUNIT. Is `behave` present on `java_handler` or outside that ?

Reply:
@pkajaba  Added more asserts to the test now.